src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay/additional-information.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: "Additional information"
+metaDescription: "Data consumption, User privacy and security concerns."
+freshnessValidatedDate: 2024-12-19
+---
+
+## Data consumption [#data-consumption]
+
+Session replay follows the same consumption pricing as your other browser bytes. The amount of bytes produced depends on the count, length, and user-activity levels of sessions, as well as the complexity of your site's DOM.
+
+The most accurate way to project your cost per replay is to enable the feature for a short test period and measure your actual consumption. Alternatively, you can use an average of 5.3 MB per replay as a rough approximation, though keep in mind that your actual cost may vary significantly. A simple formula to project your approximate costs is: `sessions` x `sampling_rate` x `gb_per_replay` x `cost_per_gb`. Here's an example:
+
+```
+1 million (sessions) x 5% (sampling_rate) x 0.0053 (gb_per_replay) x $0.35 (cost_per_gb) = $92.75 for 50,000 replays
```

Comment:
Why is this formatted as code with the copy functionality?

Reply:
I have updated this information